facebook/react-native
# [ListView] Reset ListViewDataSource "dirty bit" so that a row will only ever re-render once when changed

Currently, a ListView will continue to re-render dirty rows until the ListViewDataSource is updated with new data.  See more details here: https://github.com/facebook/react-native/issues/5255

**Test plan (required)**

1) Log to console each time a row renders
2) Update datasource for a single row
3) Check that the row only ever re-renders once
